### PR TITLE
Centralize axios configuration for different requests

### PR DESCRIPTION
**Pull request by florenperetti**
https://escuelatech.atlassian.net/browse/PW-10

- Separated axios configurations according to baseURL
- Create services files that uses each configuration accordingly. Each file corresponds to an endpoint (eg: /api/user/)

### CODE REVIEW

File: pranama-ui/src/services/malabar/axios.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+const apiClient = axios.create({
```
Comment (escuelatech) [inline pranama-ui/src/services/malabar/axios.js:3]: Where are you using this function in the code ? I see there are two axios.js

Comment (florenperetti) [reply pranama-ui/src/services/malabar/axios.js:3]: Each /axios.js configures a custom axios instance that are used in the services, eg in `pranama-ui/src/services/api/UserService.js` in line 1 you are importing that configured axios instance

File: pranama-ui/src/services/malabar/EmailService.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import apiClient from "./axios";
+
```
[the remark this answers is not in the document]
Comment (florenperetti) [reply pranama-ui/src/services/malabar/EmailService.js:2]: yes, here `pranama-ui/src/components/View/services/email_service/ContactPage.vue`

File: pranama-ui/src/components/View/services/email_service/ContactPage.vue [hunk 3]
```diff
@@ -50,15 +50,8 @@
...
-const apiClient = axios.create({
-    baseURL: 'http://malabarhangouts.com',
-    withCredentials: false,
-    headers: {
-        Accept: 'application/json',
-        'Content-Type': 'application/json'
-    }
-})
+
+import EmailService from "@/services/malabar/EmailService";
```
Comment (florenperetti) [inline pranama-ui/src/components/View/services/email_service/ContactPage.vue:54]: @escuelatech  this file is using the malabar base url https://github.com/escuelatech/Pranama/pull/5#discussion_r416247750

File: pranama-ui/src/components/View/services/login_service/LoginForm.vue [hunk 4]
```diff
@@ -98,17 +98,7 @@
...
-  baseURL: `${API}`,
-  withCredentials: false,
-  headers: {
-    Accept: "application/json",
-    "Content-Type": "application/json"
-  }
-});
+import ContactService from "@/services/api/ContactService";
 
 export default {
```
Comment (escuelatech) [inline pranama-ui/src/components/View/services/login_service/LoginForm.vue:103]: axios.js should go to the shared folder ( which is good ) . Other API services calls can you move to a folder apiservices under src. ie src/apiservices.

File: pranama-ui/src/services/api/ContactService.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import apiClient from "./axios";
```
Comment (escuelatech) [inline pranama-ui/src/services/api/ContactService.js]: API services calls, can you move to a folder apiservices under src. ie src/apiservices.

Comment (escuelatech) [reply pranama-ui/src/services/api/ContactService.js]: @florenperetti  hope I'm not confusing you

Comment (florenperetti) [reply pranama-ui/src/services/api/ContactService.js]: all good! fixed in commit 24e2b49

File: pranama-ui/src/components/View/services/login_service/LoginForm.vue (hunk 4 above)
Comment (florenperetti) [reply pranama-ui/src/components/View/services/login_service/LoginForm.vue:103]: fixed in 24e2b49

**Comment by escuelatech** [comment]
Axios as a common service

**Pull request merged**